docs/requirements-specification.md

```diff
@@ -21,4 +21,42 @@ The use cases fulfill the goals in the following way:
 | UC5 | G4, G8 |
 | UC6 | |
 
-The goals G10-G12 are currently not addressed by any use case.
\ No newline at end of file
+The goals G10-G12 are currently not addressed by any use case.
+
+### UC3: Get capability overview
+| UC3 | Get capability overview
+|---|---|
+| Actors | (Senior) Researcher |
+| Pre-conditions | The researcher is authenticated |
```

Comment:
I would require authentication. Only UC4 (identifying gaps) should be inaccessible to unauthenticated users. Instead, specify in the steps below that this UC only visualizes capabilities above a certain (configurable) evidence threshold.